### PR TITLE
Intel.PresentMon.Beta version 0.5.0.0

### PR DESCRIPTION
**Pull request by sanyer**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/117720)

### CODE REVIEW

File: manifests/i/Intel/PresentMon/Beta/0.5.0.0/Intel.PresentMon.Beta.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Intel.PresentMon.Beta
+PackageVersion: 0.5.0.0
+PackageLocale: en-US
+Publisher: Intel
```
Comment (Trenly) [inline manifests/i/Intel/PresentMon/Beta/0.5.0.0/Intel.PresentMon.Beta.locale.en-US.yaml:7]:
```suggestion
Publisher: Intel Corporation
```
This should match Control Panel

File: manifests/i/Intel/PresentMon/Beta/0.5.0.0/Intel.PresentMon.Beta.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Intel.PresentMon.Beta
+PackageVersion: 0.5.0.0
+PackageLocale: en-US
+Publisher: Intel
+PackageName: PresentMon Beta
```
Comment (Trenly) [inline manifests/i/Intel/PresentMon/Beta/0.5.0.0/Intel.PresentMon.Beta.locale.en-US.yaml:8]:
```suggestion
PackageName: Intel PresentMon
```
This should match control panel

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
Manual Validation ended with:
image

Does this package have a dependency?

File: manifests/i/Intel/PresentMon/Beta/0.5.0.0/Intel.PresentMon.Beta.locale.en-US.yaml (hunk 2 above)
Comment (sanyer) [reply manifests/i/Intel/PresentMon/Beta/0.5.0.0/Intel.PresentMon.Beta.locale.en-US.yaml:8]: I just used the other Intel's software example, but sure.

**Comment by sanyer** [comment]
@stephengillie indeed, seems like it's dependent on Visual Studio 2015, 2017, 2019, and 2022

**Review by stephengillie — APPROVED** [review]

**Pull request merged**